documentation-site/pages/blog/responsive-web/index.mdx

```diff
@@ -64,28 +64,26 @@ In many cases, you would want to override Base Web’s breakpoints with your own
 To override these, we recommend creating your own mapping, as well as generating the respective `min-width` media query strings so we can target different screen widths for styling:
 
 ```js
-const MyBreakpoints = {
+import {getMediaQuery} from 'baseui/helpers/responsive-helpers';
```

Comment:
I removed this from the blog post recently: https://github.com/uber/baseweb/pull/2159 because we don't want to direct towards importing from files deeper than one level. Those files are not considered part of the versioned api. Can you please either leave this unchanged or export the function at a directory level? Can you share why this warrants it's own function since it mostly prepends `@media screen and`? Maybe I'm missing some of it's usefulness

Reply:
I guess it’s not super useful. Before we standardized our breakpoints on our team, we had a few different variants and it was a more useful helper function then, and I just copied it over to Base Web. I’ll remove from the blog and maybe send another PR to remove it in general.